src/app/src/components/FilterSidebarSearchTab.jsx

```diff
@@ -235,14 +239,20 @@ function FilterSidebarSearchTab({
                     <InputLabel htmlFor={FACILITIES} className="form__label">
                         <FeatureFlag
                             flag="ppe"
-                            alternative="Search a Facility Name or OAR ID"
+                            alternative={
+                                embed ? textSearchLabel : DEFAULT_SEARCH_TEXT
+                            }
                         >
                             Search a Facility Name, OAR ID, or PPE Product Type
                         </FeatureFlag>
                     </InputLabel>
                     <TextField
                         id={FACILITIES}
-                        placeholder="Facility Name or OAR ID"
+                        placeholder={
```

Comment:
The placeholder text is the same as the text search label when it is embed mode and not default text.